README.md

```diff
@@ -28,15 +28,6 @@ console.log(currentDateTime);
 // Output: 2023-06-20 12:22:20
 ```
 
-## isSet
-
-This function checks if the value provided is none of this - null, undefined, empty string, "undefined", empty array as string.
-
-```bash
-console.log(isSet("null"));
-// Output: false
-```
-
 ## isSetObject
```

Comment:
Remove isSetObject instead of isSet @Madhukesh007

Reply:
@jainarpit014 
Okay